cmake/pico_pre_load_toolchain.cmake

```diff
@@ -1,6 +1,6 @@
 # PICO_CMAKE_CONFIG: PICO_TOOLCHAIN_PATH, Path to search for compiler, default=none (i.e. search system paths), group=build
-# Set your compiler path here if it's not in the PATH environment variable.
-set(PICO_TOOLCHAIN_PATH "" CACHE INTERNAL "")
+# Set your compiler path here with PICO_TOOLCHAIN_PATH CMake option, if it's not in the PATH environment variable.
```

Comment:
I'd remove this comment completely - the PICO_CMAKE_CONFIG comment explains what the var is for.

Reply:
Done